### DISCUSSION TITLE
[Q&A] 本地运行lobechat报错

### DISCUSSION THREAD

**Discussion by scortLi** [lobehub/lobehub#1024] [General Question] [answered]
### 💻 Operating System

Windows

### 📦 Environment

Official Preview

### 🌐 Browser

Chrome

### 🐛 Bug Description

按照官方给的部署顺序流程启动时报错，但是貌似不影响使用

### 📷 Recurrence Steps

$ git clone https://github.com/lobehub/lobe-chat.git
$ cd lobe-chat
$ bun install
$ bun run dev

### 📝 Additional Information

![企业微信截图_17050284234118](https://github.com/lobehub/lobe-chat/assets/14088043/9882cc21-05c1-4ffd-8a5d-89cfce97ce97)
![企业微信截图_17050281908225](https://github.com/lobehub/lobe-chat/assets/14088043/11a56c9d-24f6-4ec5-945c-41148f7d9d5a)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
你现在这个是 dev 态，会报错一次。 参考 https://github.com/lobehub/lobe-chat/discussions/873 。如果想不报错的话，请使用 

```
bun run build
bun run start
```
